tests/test_form.py

```diff
@@ -98,35 +99,27 @@ def text_callback(request, context):
         mock.register_uri('POST', url + '/post', text=text_callback)
     return mechanicalsoup.StatefulBrowser(requests_adapters={'mock': mock}), url
 
-def test_choose_submit():
-    browser, url = setup_mock_browser(
-        expected_post=[
+@pytest.mark.parametrize("expected_post", [
+    pytest.param(
```

Comment:
Travis-CI says "AttributeError: 'module' object has no attribute 'param'" for Python 2.6, 3.3 and 3.4.

Any way to get this to work for all Python versions? Not sure whether it would work, but perhaps just adding a "pip install --update pytest" in .travis.yml could do the trick?

Reply:
Thanks for the suggestion! Using `pip install --upgrade pytest` provides a sufficiently recent version of pytest.